server/views/pages/prisonerList/index.njk

```diff
@@ -24,3 +24,5 @@
   </div>
 
 {% endblock %}
+
+{% block mojFrontendInit %}{% endblock %}
```

Comment:
On the prisoner list page, we use a custom version of the sortable table (which we took from the CIAG UI), which includes our own markup and logic for sorting.

If you initialise the MoJ frontend JavaScript, it changes the behaviour of our custom version, so this empty {% block ... %} basically stops the MoJ frontend JavaScript initialising block from the `layout` by replacing it with empty content.

There may be a better approach to this, but best I could think of for now!